aspnetcore/getting-started-1.1.md

```diff
@@ -23,9 +23,6 @@ uid: getting-started-1.1
 
 1. Install [.NET Core](https://microsoft.com/net/core).
 
-   If you're on Windows, select the **Command line / other** environment. 
```

Comment:
@Rick-Anderson That's how I refer to it. Just want to make sure we're all on the same page.